src/order/conditionally_complete_lattice.lean

```diff
@@ -537,8 +537,8 @@ end conditionally_complete_lattice
 section conditionally_complete_linear_order
 variables [conditionally_complete_linear_order α] {s t : set α} {a b : α}
 
-/--When b < Sup s, there is an element a in s with b < a, if s is nonempty and the order is
-a linear order.-/
+/-- When b < Sup s, there is an element a in s with b < a, if s is nonempty and the order is
+a linear order. -/
 lemma exists_lt_of_lt_cSup (_ : s ≠ ∅) (_ : b < Sup s) : ∃a∈s, b < a :=
```

Comment:
Why not give names to the hypotheses? That will guarantee that the theorem looks nice when users have to see it.

Reply:
More than fifty hypotheses in this file have no name. Maybe it's better to add names in another PR.